src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,192 @@
+package parser;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+// Import Commands
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+import command.UpdateCommand;
+// Import Luna Exceptions
+import exception.LunaException;
+import exception.LunaInvalidCommandException;
+import exception.LunaInvalidDateException;
+import exception.LunaInvalidDescriptionException;
+import exception.LunaInvalidIndexException;
+// Import Tasks;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+
+/**
+ * Deals with making sense of the user command.
+ *
+ * @author fannyjian
+ */
+public class Parser {
+
+    public static String getCommandWord(String command) {
+        return command.split(" ", 2)[0];
+    }
+
+    /**
+     * Takes in a string of user command and creates a new instance of Command
+     * according to the keyword.
+     *
+     * @param cmd String of command to process.
+     * @return Command corresponding to the user's instruction.
+     * @throws LunaException.
+     */
+    public static Command parse(String cmd) throws LunaException {
+        String[] cmdSplit = cmd.split(" ", 2);
+        Command c;
+        switch (cmdSplit[0]) {
+            case "todo":
```

Comment:
Perhaps you can change the indentation of the case statement to the same as the switch statement?